tutorials/cloud-iot-gateways-rpi/index.md

```diff
@@ -0,0 +1,272 @@
+
+9. Leave everything else as-is, and click **Create**.
+
+[create-registry]: ./images/create-registry.png 
+
+## Set Up your Gateway
+
+For the purposes of this tutorial, you will use your laptop or desktop as the gateway device. You will first generate an RSA public/private key pair, which will be used to sign the JWTs for authenticating to Cloud IoT Core.
+
+To set up your gateway:
+
+1. Clone the following repository and change into the directory for this tutorial's code:
+
+        git clone https://github.com/GoogleCloudPlatform/community.git
+        cd community/tutorials/cloud-iot-gateways-pi
+
+2. Generate an RS256 public/private key pair by running the following:
+
+        ./generate_keys.sh
+
+3. In the [Cloud IoT Core console][cloud-iot], click on the registry you created.
+4. Under the **Gateways** tab, click **Create Gateway**.
+5. For **Gateway ID**, enter `my-gateway`.
+6. Copy the contents of `rsa_public.pem` into the public key text area.
+7. For **Device authentication method**, select **Association only**. For more details about why this option is used, see [Extra Notes][extra-notes] below.
+
+    ![create gateway][create-gateway]
+
+8. Click **Create**.
+9. Export your Google Cloud project ID as an environment variable by running the following:
+
+        export GOOGLE_CLOUD_PROJECT=your-project-id-123
+
+10. Modify the `run-gateway` script by providing arguments for `registry_id` and `device_id` if you chose different names.
+11. Download [Google's CA root certificate](https://pki.goog/roots.pem) into the same directory if it doesn't exist already.
+
+        wget https://pki.goog/roots.pem
+
+12. Run the following command to start the gateway:
+
+        source run-gateway
+
+13. Keep this process running while you proceed through the next steps. We recommend that you use a new tab or window for each gateway and device.
+
+14. Find the local IP address of the gateway using `ifconfig` on  Mac/Linux or `ipconfig /all` on Windows. Copy this somewhere as you will need to add this IP address to `led-light.py` and `thermostat.py` later.
+
+[extra-notes]: #extra-notes
+[create-gateway]: ./images/create-gateway.png
+
+## Raspberry Pi Setup
+
+In this tutorial, you'll use a [Raspberry Pi][rpi] to manage the LED/temperature sensor. Devices will connect to the gateway device through [UDP sockets][udp-socket], which will connect to Cloud IoT Core via the [MQTT bridge][mqtt-bridge]. A Raspberry Pi could theoretically connect directly to the cloud (since the Pi can connect to the internet), so using a Raspberry Pi for this part is purely for demonstration purposes.
+
+1. [Download Raspian][raspbian-download] (the full image with Desktop and recommended software) and follow [the installation guide][raspbian-installation] to flash Raspbian onto your microSD card.
+2. Insert the microSD card with Raspbian into your Raspberry Pi.
+3. Attach a power source to the Raspberry Pi using the microUSB cable (i.e. to a laptop USB port).
```

Comment:
Do you mean 'e.g.' (which means "for example") or do you mean 'i.e.' (which means "that is")?

Reply:
Whoops, yes, will change to 'e.g'